docs/fastdds/xml_configuration/dynamic_types.rst

```diff
@@ -54,30 +54,30 @@ For each of the types detailed below, an example of how to build the type's XML
 Primitive types
 """""""""""""""
 
-Primitive types should be declared as members of an aggregated type (`Structure types`_ or `Union types`_).
-Primitive types are defined by attribute :code:`type` and the possible values are listed in the table below.
+Primitive types are built-in types and they should be declared as members of an aggregated type (`Structure types`_ or
```

Comment:
I suggested to use `declare` instead of `define` in primitive types, because there are built-in types and they are already defined. But in the rest of types, the user is defining those types.